ts/screens/profile/ProfileMainScreenTopBanner.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Alert,
+  Banner,
+  ContentWrapper,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import React from "react";
+import { setShowProfileBanner } from "../../features/profileSettings/store/actions";
+import { profileBannerToShowSelector } from "../../features/profileSettings/store/selectors";
+import { openSystemNotificationSettingsScreen } from "../../features/pushNotifications/utils";
+import I18n from "../../i18n";
+import { useIONavigation } from "../../navigation/params/AppParamsList";
+import ROUTES from "../../navigation/routes";
+import { useIODispatch, useIOSelector } from "../../store/hooks";
+
+export const ProfileMainScreenTopBanner = () => {
+  const bannerToShow = useIOSelector(profileBannerToShowSelector);
+  const dispatch = useIODispatch();
+  const navigation = useIONavigation();
+
+  const handleCloseBanner = () => dispatch(setShowProfileBanner(false));
```

Comment:
In order to optimize code you can use useCallback 
```suggestion
  const handleCloseBanner = useCallback(
    () => dispatch(setShowProfileBanner(false)),
    [dispatch]
  );
```